health/micro-ui/web/packages/modules/campaign-manager/src/pages/employee/NewAppConfiguration/components/DropdownTemplate.js

```
@@ -1,11 +1,34 @@
+import { FieldV1, Loader } from "@egovernments/digit-ui-components";
 
 const DropdownTemplate = ({ field, t, fieldTypeMasterData, isFieldSelected, props }) => {
     const selectedField = field || props?.field;
     const fieldType = fieldTypeMasterData || props?.fieldTypeMasterData;
     const selected = isFieldSelected || props?.isFieldSelected;
 
+    // Check if MDMS is enabled
+    const isMdmsEnabled = !!selectedField?.isMdms && !!selectedField?.schemaCode;
+
+    // Fetch MDMS data if schemaCode exists
+    const { isLoading, data } = Digit?.Hooks.useCustomMDMS(
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)